app/controllers/v1/data_types_controller.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module V1
+  # See the DataTypeResource for more information
```

Comment:
Might be worth providing a bit more explanation on data types. I can see what it is doing but not quite sure of it's significance.